interface/js/app/history.js

```diff
@@ -117,7 +117,7 @@ function($, _, Humanize) {
             preprocess_item(item);
             Object.keys(item.symbols).map(function(key) {
                 var sym = item.symbols[key];
-                if (!sym.name) {
+                if (!sym.name || sym.name == "undefined") {
```

Comment:
But why is it undefined? Maybe there is a better check for a missing hash field than this ugly comparison?

Reply:
You're right, I think I understood where it's going replaced by "undefined" string. I will close this PR and open another one